.github/workflows/container_build.yml

```diff
@@ -0,0 +1,74 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build and deploy containers
+
+# Controls when the action will run. 
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - master
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+
+      - name: Check Out Repo 
+        uses: actions/checkout@v2
+
+      - 
+        name: Docker meta
```

Comment:
should the `-` not be on the same line as the `name:`. It just looks a bit odd otherwise

Reply:
Thanks, merged this, not sure how it got missed!